wire-ios/WireUITests/PersonalUserTests.swift

```diff
@@ -24,26 +24,50 @@ final class PersonalUsersTests: WireUITestCase {
     func test_Register_asPersonalUser() async throws {
         let user = UserGenerator.generateUniqueUserInfo()
 
-        let page = WelcomePage()
+        let welcomePage = try WelcomePage()
+            .assertHasLoaded()
+
+        let createAccountPage = try welcomePage
             .typeEmailOrSSO(user.email)
             .tapCreatePersonalAccountLink()
+            .assertHasLoaded()
+
+        let verificationPage = try createAccountPage
             .tapConfirmCreateAccount()
             .tapAcceptButton()
+            .assertHasLoaded()
 
         let verificationCode = try await InbucketClient.getVerificationCode(email: user.email)
 
-        let accountPage = page
+        let setNamePage = try verificationPage
             .enterVerificationCode(verificationCode)
+            .assertHasLoaded()
+
+        let setPasswordPage = try setNamePage
             .setName(user.name)
+            .assertHasLoaded()
+
+        let setUsernamePage = try setPasswordPage
             .setPassword(user.password)
             .acceptPopup()
+            .assertHasLoaded()
+
+        let conversationsPage = try setUsernamePage
             .setUsername(user.username)
+            .assertHasLoaded()
```

Comment:
Was trying to get away from explicitly typing this every time - is this just not possible with XCTest?

Reply:
hm, ok I move the assert call to the init of Page so initialize a page is now throwing. So it will stop the test and you don't have to have this explicit call